src/Creator.tsx

```diff
@@ -159,27 +152,31 @@ export const Creator = ({ NFToupon_Key }: Props) => {
   };
 
   const sendDetails = async () => {
-    const response = await fetch(
-      `https://eatozee-crypto.app/api/nftoupon/creator/mint`,
-      {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-          'NFToupon-Key': NFToupon_Key,
-        },
-        body: JSON.stringify({
-          file: src,
-          address: walletAddress,
-        }),
-      }
-    );
-    const { payload } = await response.json();
-
-    if (!isEmpty(payload)) {
-      setXummPayload(payload);
-      setVisible(true);
+    if (isEmpty(textAreaValue.trim())) {
+      setValidation(true);
     } else {
-      setXummPayload(null);
+      const response = await fetch(
+        `https://eatozee-crypto.app/api/nftoupon/creator/mint`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+            'NFToupon-Key': NFToupon_Key,
+          },
+          body: JSON.stringify({
+            file: src,
+            address: walletAddress,
+          }),
+        }
+      );
+     
+      const { payload } = await response.json();
+      if (!isEmpty(payload)) {
+        setXummPayload(payload);
+        setVisible(true);
+      } else {
+        setXummPayload(null);
```

Comment:
is there any reason we avoided using toast ?
the way you have implemented in acceptOffer function?

Reply:
No particular reason will do it right away